third_party/txt/src/txt/font_collection.cc

```diff
@@ -126,8 +126,13 @@ FontCollection::GetMinikinFontCollectionForFamily(
         minikin_family,
     };
     if (enable_font_fallback_) {
-      for (std::string fallback_family : fallback_fonts_for_locale_[locale])
-        minikin_families.push_back(fallback_fonts_[fallback_family]);
+      for (std::string fallback_family : fallback_fonts_for_locale_[locale]) {
+        std::shared_ptr<minikin::FontFamily> fallback_font_ =
```

Comment:
Can you change this so that it will not insert an entry for the missing family into `fallback_fonts_`?

For example:
```
      for (std::string fallback_family : fallback_fonts_for_locale_[locale]) {
        auto it = fallback_fonts_.find(fallback_family);
        if (it != fallback_fonts_.end()) {
          minikin_families.push_back(it->second);
        }
      }
```